.github/workflows/docker-build-and-test.yml

```diff
@@ -30,36 +30,61 @@ permissions:
   packages: write
 
 jobs:
+  #=============================
+  # Dynamically generate matrix
+  #=============================
+
+  get-matrix:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: get-matrix
+        id: get-matrix
+        run: |
+          # Disable SC2046: Quote this to prevent word splitting
+          # I can't quite this, just look at it
```

Comment:
I think there is either a word missing here or the `e` is too much in quite

Reply:
Typo :)